truckersmp-cli

```diff
@@ -430,7 +430,7 @@ def start_with_wine():
     env["WINEARCH"] = "win64"
     env["WINEPREFIX"] = args.prefixdir
     env["WINEDLLOVERRIDES"] = "d3d11=;dxgi=" if not args.enable_d3d11 else ""
-    argv = ["wine", ]
+    argv = [wine, ]
```

Comment:
What's that empty second array member for?
Just curious, maybe that's a python thing I don't know about.

Reply:
Since this is not a [tuple](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences) but a list, we can replace it with `argv = [wine]`.

* `[1, ]` or `[1]` is a one-element list.
* `(1, )` is a one-element tuple.

But `(1)` is an integer because it doesn't contain `,`.

```python
a = [1, ]
b = [1]
c = (1, )
d = (1)

print("a:", a)  # "[1]"
print("b:", b)  # "[1]"
print("c:", c)  # "(1, )"
print("d:", d)  # "1" (Not a tuple but an integer)
```

So I generally write in the format `(elem, )`/`[elem, ]` for both one-element tuples and *one-element lists*.